catalog/ui/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.hbs

```diff
@@ -27,23 +27,23 @@
         <span>
             Clear
         </span>
-    </button>
-    <button class="footer-start is-positive">
+    </button><!--
+    --><button class="footer-start is-positive">
```

Comment:
Remove empty HTML comment

Reply:
These actually serve a purpose. The handlebars template inserts whitespace if you put the button on a new line and don't do this.

Reply:
@adimka Ah, cool, good to know.

Reply:
Basically exactly what Adam said, there are other ways to get around this but this is the best I've found.  A good portion of this PR I went through and removed the other way of doing it (search for whitespaceFix).